### DISCUSSION TITLE
Is it possible to define a specific path for Sonatype (releases) badges?

### DISCUSSION THREAD

**Discussion by Andre601** [badges/shields#5386] [Q&A] [answered]
I want to display the latest available release version that is on a Nexus Repository Manager using the `Sonatype Nexus (releases)` badge.
But doing so displays the latest dev build that is on the Nexus (And is also displayed as such (orange color)) which is not what I want.

I also tried to use the Repository one which temporarily fixed that issue but it's now back again.
The dev builds are in a separate directory from the main releases, yet the badge seems to just display the latest version, no matter if it is a dev version or not.

Is there any way to force Shields.io to use a specific path and not use any other one?

**Comment by calebcartwright** [comment]
Could you provide some additional details, namely the badge url you're attempting to use for your project?

**Reply by Andre601** [reply]
I tried two types:
- https://img.shields.io/nexus/r/me.clip/placeholderapi?server=https%3A%2F%2Frepo.extendedclip.com ![](https://img.shields.io/nexus/r/me.clip/placeholderapi?server=https%3A%2F%2Frepo.extendedclip.com)
- https://img.shields.io/nexus/placeholderapi/me.clip/placeholderapi?server=https%3A%2F%2Frepo.extendedclip.com ![](https://img.shields.io/nexus/placeholderapi/me.clip/placeholderapi?server=https%3A%2F%2Frepo.extendedclip.com)

Interestingly enough does the second URL now display the right badge, while it didn't a few hours ago.

**Comment by Andre601** [comment, marked as answer]
Seems to just be an issue where the original badge was reset to the old one for some reason...
